packages/vm/src/runBlock.ts

```diff
@@ -968,11 +969,16 @@ export class ValidatorWithdrawalRequest extends CLRequest implements CLRequestTy
  * @param _vm VM instance from which to derive CL requests
  * @returns an list of CL requests in ascending order by type
  */
-export const accumulateRequests = async (vm: VM): Promise<CLRequest[]> => {
+export const accumulateRequests = async (
+  vm: VM,
+  txResults: RunTxResult[]
+): Promise<CLRequest[]> => {
   const requests: CLRequest[] = []
   const common = vm.common
 
-  // TODO: Add in code to accumulate deposits (EIP-6110)
+  if (common.isActivatedEIP(6110)) {
+    await accumulateDeposits(txResults, requests)
```

Comment:
accumulate7002 is with `_` while this is not, may be we take a consistent approach